<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorPickerColor</class>
 <widget class="QWidget" name="ColorPickerColor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>122</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>8</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>8</number>
     </property>
     <property name="bottomMargin">
      <number>4</number>
     </property>
     <property name="spacing">
      <number>4</number>
     </property>
     <item row="1" column="2">
      <widget class="ScrSpinBox" name="numberAlpha">
       <property name="suffix">
        <string> %</string>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>10.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="labelAlpha">
       <property name="text">
        <string>Opacity:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="labelShade">
       <property name="text">
        <string>Shade:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="ScrSpinBox" name="numberShade">
       <property name="suffix">
        <string> %</string>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>10.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="SectionContainer" name="sectionSwatches">
     <property name="title" stdset="0">
      <string>Color Swatches</string>
     </property>
     <widget class="ColorPickerColorSwatches" name="swatches"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SectionContainer</class>
   <extends>QWidget</extends>
   <header>ui/widgets/section_container.h</header>
   <addpagemethod>setWidget</addpagemethod>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ScrSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header location="global">ui/scrspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>ColorPickerColorSwatches</class>
   <extends>QWidget</extends>
   <header>ui/colorpicker/colorpicker_color_swatches.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>numberShade</tabstop>
  <tabstop>numberAlpha</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
